networking/datasources.tf

```diff
@@ -0,0 +1,16 @@
+data "tfe_outputs" "dev" {
```

Comment:
Doesn't this not actually solve the problem? Here, this data source still depends on another workspace's resources.

The idea I shared today was more meant to query the underlying data directly.

e.g Instead of `data.tfe_outputs.live.values.teleport_internal_dns_name` one would do something like `data.google_dns_record_set.teleport_internal.name` (after declaring that data source of course).